project_one.css

```diff
@@ -0,0 +1,174 @@
+.container {
+  display: grid;
+  grid-template-columns: auto;
+  padding: 10px;
+  grid-column-start: 1;
+  grid-auto-rows: max-content;
+  row-gap: 50px;
+}
+.card {
+  left: 24px;
+  top: 114px;
+  border-radius: 16px;
+  padding: 38px;
+  text-align: left;
+  justify-content: center;
+}
+.snap {
+  display: flex;
+  justify-content: center;
+}
+.main {
+  padding: 10px;
+  font-family: "Poppins", sans-serif;
+}
+h1 {
+  color: #172b4d;
+  font-size: 32px;
+  font-weight: bold;
+}
```

Comment:
- [ ] Good job by using the grid to layout the container of the work section but it would be better if you also use flex to align the hole card container, not just the image container 
![image](https://user-images.githubusercontent.com/74744787/159126742-f1484b74-2752-48a6-af31-7f87561e729f.png)